pkg/runner/k6-summary.txt

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Looks like some test output (add to ignore or remove)

Reply:
This file is required by the unit test to check proper test result output parsing.